iot/bpi96/README.md

```diff
@@ -5,9 +5,11 @@ permalink: /documentation/iot/bpi96/
 
 # Using the BPi96
 
+<p><font color="red" size="5">NOTE: THE I/O VOLTAGE ON THIS BOARD IS 3.3V AND IT IS NOT COMPATIBLE WITH 1.8V MEZZANINES OR PERIPHERALS.</font></p>
```

Comment:
Thanks Sahaj.

To be honest I'd get even more assertive and substitute "1.8V mezzanines" with "any 96Boards mezzanines or peripherals not *explicitly* designed for 3.3V tolerance".

This connector is not 96Boards, no 96Boards mezzanine will work with it. Even trying to design a mezzanine that could be compatible with both 96Boards LS and BPi-NBIoT LS is pretty hard (there's no I/O reference voltage on BPi-NBIoT to power any level shifters so I can't really see any way to automatically detect the baseboard I/O voltage).

Reply:
ack. will change